recipe/meta.yaml

```diff
@@ -13,8 +13,7 @@ source:
   - patches/corei5.patch
 
 build:
-  number: 0
-  skip: True  # [win and vc<14]
```

Comment:
Can you keep this here? No need to remove it

Reply:
Yup. This was for testing.